packages/plugin-core/assets/dendron-ws/tutorial/tutorial.4-images-and-diagrams.md

```diff
@@ -0,0 +1,78 @@
+---
+id: vAy0awBTwDaxKGHWUdSaP
+title: Images and Diagrams
+desc: ''
+updated: 1624452350303
+created: 1624352425213
+---
+After completing this section you'll know how to create rich content within your Markdown notes.
+
+### Images
+
+As you may have already noticed already in the tutorial pages, you can insert images into your notes. 
+
+> 🌱 The easiest way to do this is to first download the [Paste Image](https://marketplace.visualstudio.com/items?itemName=mushan.vscode-paste-image) extension. Once installed, copy the image onto your clipboard, and then use the `Paste Image` command into your editor pane.
+
+This will automatically create a link for you and copy the file contents into the `\assets\images\` directory in your workspace.
```

Comment:
path separators are os specific. not sure how to best show this. i would default to forward slashes since that's more common, documentation wise

Reply:
Just re-worded this to say `assets` directory.